src/manifest.json

```diff
@@ -16,6 +16,12 @@
         "/post_menu",
         "/in_post"
     ],
+    "on_install": {
+        "url": "/install"
+    },
+    "bindings": {
+        "url": "/bindings"
+    },
```

Comment:
Declaring the path here isn't strictly needed as it's the default path anyway, but I found it easier to understand why a certain path is called. 1/5